doc/browsing_cubes/overlay.ipynb

```diff
@@ -47,15 +65,6 @@
     "\n",
     "Browser([plot1, plot2]).display()"
    ]
-  },
```

Comment:
It's not in the range of the changes you've made to this notebook, but it doesn't work because `cube` doesn't exist when setting the default projection just above (L40 / 58).